README.md

```diff
@@ -56,6 +56,7 @@ Also, check the [standard repository of TLA+ examples][].
 
 ## Community
 
+- Discuss the use of Apalache and R&D on the [Apalache Forum][].
 - Join the chat in the [Apalache zulip stream].
```

Comment:
And we keep zulip? Won't it confuse people?

Reply:
yeah, let's maybe remove it entirely?